aws/coreos-docker-servers.tf

```diff
@@ -30,3 +33,23 @@ resource "template_file" "etcd_discovery_url" {
     }
   }
 }
+
+resource "aws_security_group" "docker_node" {
```

Comment:
I'm not sure it can be separated. How would you do it? You mean restricting router to only 1024-4242 and 4244-65535?